packages/usa-modal/src/index.js

```diff
@@ -87,11 +80,11 @@ function toggleModal(event) {
     // If it doesn't have an ID, make one
     // Store id as data attribute on modal
     if (clickedElement.hasAttribute(OPENER_ATTRIBUTE)) {
-      if (this.getAttribute("id") === null) {
+      if (clickedElement.getAttribute("id") === null) {
```

Comment:
I started getting an error related to `this` once I passed a parameter to the click event in line 308. I updated this to `clickedElement` and wanted to confirm that this was the correct replacement. I see that it still creates and assigns an opener `id` as needed.